src/reducers.js

```diff
@@ -12,7 +12,7 @@ export const reducer = (
     announcer: null,
   },
   action) => {
-    console.log(action);
+    // console.log(action);
```

Comment:
You should leave this uncommented. It's a valuable source of info, and is high-level enough that it makes sense to always print.